pkg/app/app.go

```diff
@@ -42,7 +42,7 @@ import (
 	"github.com/automationbroker/bundle-lib/registries"
 	agnosticruntime "github.com/automationbroker/bundle-lib/runtime"
 	"github.com/automationbroker/config"
-	"github.com/jessevdk/go-flags"
+	flags "github.com/jessevdk/go-flags"
```

Comment:
Does this not cause issues?

Reply:
interesting goimports changed this, On an old version of goimports, it completely removed it, and on the other machine it added the above. I'll fix it using gedit :) But no it didn't seem to cause any issues since we're not also importing the system flags.

Reply:
So goimports won't touch it now that I fixed it. It's weird that `go-flags` is actually used as `flags.`